tc-tests-utils.sh

```diff
@@ -390,6 +471,8 @@ is_patched_bazel()
 {
   bazel_version=$(bazel version | grep 'Build label:' | cut -d':' -f2)
 
+  bazel shutdown
```

Comment:
Any reason why you're not using `shutdown_bazel` here?

Reply:
Yes, `shutdown_bazel` ensures using the bazel output directory, if we issue a `bazel shutdown` with a wrong path (i.e., it should be symetric to the `bazel` call that started the process) then it will leave dangling processes.